perfscale_regression_ci/scripts/network/network_policy_scalability.sh

```diff
@@ -0,0 +1,149 @@
+check_http_code $deny_traffic_code $pod_ip $apiserver_pod
+deny_time_end=`date +%s`
+deny_final_time=$((deny_time_end - deny_time_start))
+echo "Time taken for network policy to become inactive: $deny_final_time s"
+
+sleep 1
+
+echo "Starting test to allow network traffic in the namespace ${NAMESPACE}..."
+oc delete networkpolicy deny-by-default -n ${NAMESPACE}
+return_traffic_start=`date +%s`
+check_http_code $return_traffic_code $pod_ip $apiserver_pod
+return_traffic_end=`date +%s`
+return_traffic_final_time=$((return_traffic_end - return_traffic_start))
+echo "Time taken for network policy to become active: $return_traffic_final_time s"
+
+echo "=========================================================================="
+
+policy_no_policy_difference=$(calculate_difference ${final_time_np} ${final_time_no_np})
+deny_return_traffic_difference=$(calculate_difference ${deny_final_time} ${return_traffic_final_time})
+
+if [[ ( $final_time_np -le 120 ) ]]; then
+	echo "Expected: test time with network policy $final_time_np s <= 120 s (2 minutes)."
+	((++pass_fail))
+else
+	echo "Test time with network policy $final_time_np s >= 120 s (2 minutes)."
+fi
+
+
+if [[ ( $final_time_no_np -le 120 ) ]]; then
+	echo "Expected: test time without network policy $final_time_no_np s <= 120 s (2 minutes)."
+	((++pass_fail))
+else
+	echo "Test time without network policy $final_time_no_np s >= 120 s (2 minutes)."
+fi
+
+
+if [[ ( $deny_final_time -le 10 ) ]]; then
+	echo "Expected: Time when network traffic blocked $deny_final_time s <= 10 s."
+	((++pass_fail))
+else
+	echo "Time when traffic blocked $deny_final_time s >= 10 s."
+fi
+
+
+if [[ ( $return_traffic_final_time -le 10 ) ]]; then
+	echo "Expected: Time when network traffic returns $return_traffic_final_time s <= 10 s."
+	((++pass_fail))
+else
+	echo "Time when network traffic returns $return_traffic_final_time s >= 10 s."
+fi
+
+
+if [[ ( $policy_no_policy_difference -le 10 ) ]]; then
+	echo "Expected: Difference in test times (with and without network policy) $policy_no_policy_difference s <= 10 s."
+	((++pass_fail))
+else
+	echo "Difference in test times (with and without network policy) $policy_no_policy_difference s >= 10 s."
+fi
+
+
+if [[ ( $deny_return_traffic_difference -le 10 ) ]]; then
+	echo "Expected: Difference in test times (traffic denied and traffic returned) $deny_return_traffic_difference s <= 10 s."
+	((++pass_fail))
+else
+	echo "Difference in test times (traffic denied and traffic returned) $deny_return_traffic_difference s >= 10 s."
+fi
+
+echo ""
+echo "======Final test result======"
+
+if [[ $pass_fail -eq 6 ]]; then
+	echo -e "\nOverall NetworkPolicy scalability - using customer network policy Testcase result:  PASS"
+  	echo "======Clean up test environment======"
+	# # delete projects:
+  	# ######### Clean up: delete projects and wait until all projects and pods are gone
+  	echo "Deleting test objects"
+	delete_project_by_label kube-burner-job=${kube_burner_job_list[0]}
```

Comment:
here you can just use **delete_project_by_label kube-burner-job** and it should delete all projects that have any label after _kube-burner-job=*_